public/components/notebooks/components/paragraph_components/paragraphs.tsx

```diff
@@ -510,7 +520,14 @@ export const Paragraphs = forwardRef((props: ParagraphProps, ref) => {
     <>
       <EuiPanel>
         {renderParaHeader(!para.isVizualisation ? 'Code block' : 'Visualization', index)}
-        <div key={index} className={paraClass} onClick={() => paragraphSelector(index)}>
+        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+        <div
```

Comment:
"non interactive elements with `key` must also have an `onKeypress` keyboard handler`.   In this case not for us.